src/briefcase/platforms/web/static.py

```diff
@@ -447,6 +452,19 @@ class StaticWebPublishCommand(StaticWebMixin, PublishCommand):
     default_publication_channel = "s3"
 
 
+class StaticWebDevCommand(StaticWebMixin, DevCommand):
+    description = "Run a static web project in development mode. (Work in progress)"
+
+    def run_dev_app(self, app: AppConfig, env, passthrough=None, **kwargs):
+        raise UnsupportedCommandError(
+            platform="",
+            output_format="Web",
+            command="Dev",
```

Comment:
Web is the platform; the output format is static (we could, for example, deploy to docker).
```suggestion
            platform="web",
            output_format="static",
            command="dev",
```